pcsx2/SaveState.h

```diff
@@ -24,7 +24,7 @@
 //  the lower 16 bit value.  IF the change is breaking of all compatibility with old
 //  states, increment the upper 16 bit value, and clear the lower 16 bits to 0.
 
-static const u32 g_SaveVersion = (0x9A0B << 16) | 0x0000;
+static const u32 g_SaveVersion = (0x9A0C << 16) | 0x0000;
```

Comment:
I don't think you need to bump this save state version unless you've actually changed the save state format of PCSX2 itself.